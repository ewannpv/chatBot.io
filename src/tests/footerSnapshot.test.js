import React from 'react';
import renderer from 'react-test-renderer';
import FooterContainer from '../components/home/containers/footer-container';

test('footer renders correctly', () => {
  const tree = renderer
    .create(<FooterContainer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
