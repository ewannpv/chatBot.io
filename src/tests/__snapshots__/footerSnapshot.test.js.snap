// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders correctly 1`] = `
<footer>
  <nav
    className="navbar navbar-expand navbar-dark bg-dark fixed-bottom"
  >
    <div
      className="container-xl"
    >
      <div
        className="navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rb-event-key="https://github.com/ewannpv"
          href="https://github.com/ewannpv"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Github
        </a>
      </div>
    </div>
  </nav>
</footer>
`;
